server/src/test/java/org/elasticsearch/cluster/service/ClusterApplierRecordingServiceTests.java

```diff
@@ -116,4 +135,42 @@ public void testRecordingServiceStats() {
         }
     }
 
+    @TestLogging(reason = "testing debug logging", value = "org.elasticsearch.cluster.service.ClusterApplierRecordingService:DEBUG")
+    public void testSlowTaskDebugLogging() {
+        final var debugLoggingTimeout = TimeValue.timeValueMillis(between(1, 100000));
+        var recorder = new Recorder(threadPool, debugLoggingTimeout);
```

Comment:
maybe a tiny comment why these two cases in the test are different?

Reply:
++ done in f9bd56f3dc55c3b7d562fecd1bf3e29b8850c0f6.

Reply:
Thank you, it just prevents squinting too long at them! :)))